src/main/java/car/controller/RacingGameController.java

```diff
@@ -0,0 +1,54 @@
+package car.controller;
+
+import car.model.*;
+import car.view.Constant;
+import car.view.InputView;
+import car.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGameController {
+
+    private final InputView input = new InputView();
+    private final CarName carName = new CarName();
+    private RacingCars racingCars;
+    private GameNumber gameNumber;
+    private final OutputView output = new OutputView();
+    private final Winner winner = new Winner();
+    public RacingGameController() {
+        startGame();
+        output.gameResultMessage();
+        for(int i=0; i<gameNumber.getGameNumber(); i++){
+            List<String> results = playGame(racingCars);
+            output.gameStatus(results);
+        }
+        endGame(racingCars);
+        String winner = racingCars.getWinner();
+        output.gameWinner(winner);
+    }
+
+    private void startGame() {
+        String carNames = input.inputCarName();
+        carName.setCarName(carNames);
+        racingCars = new RacingCars(carName.getCarNames());
+
+        gameNumber = new GameNumber(input.inputGameNumber());
+    }
+    private List<String> playGame(RacingCars racingCars){
+        List<Car> cars = racingCars.getCars();
+        List<String> results = new ArrayList<>();
+        for (Car car : cars) {
+            car.move(RandomNumberGenerator.generate());
+            String result = car.getName() + Constant.CAR_NAME_BORDER;
+            for(int i=0; i<car.getPosition(); i++){
```

Comment:
전체적으로 Java 컨벤션 및 스타일 지켜주세요!
```suggestion
            for(int i = 0; i < car.getPosition(); i++) {
```